src/neo/SmartContract/Helper.cs

```diff
@@ -138,19 +140,25 @@ internal static bool VerifyWitnesses(this IVerifiable verifiable, StoreView snap
             if (hashes.Length != verifiable.Witnesses.Length) return false;
             for (int i = 0; i < hashes.Length; i++)
             {
+                int offset;
                 byte[] verification = verifiable.Witnesses[i].VerificationScript;
                 if (verification.Length == 0)
                 {
-                    verification = snapshot.Contracts.TryGet(hashes[i])?.Script;
-                    if (verification is null) return false;
+                    ContractState cs = snapshot.Contracts.TryGet(hashes[i]);
+                    if (cs is null) return false;
+                    ContractMethodDescriptor md = cs.Manifest.Abi.GetMethod("verify");
```

Comment:
I think that we should use a constant for this `verify`, it will be easier if we want to know what standard methods we have.

Reply:
Like this is more direct,however constant is also good.

Reply:
I can't find a good place to put the constant.

Reply:
> I can't find a good place to put the constant.

ContractManifest?

If a contract has no `verify` method, so it can't create a valid witness?
Or always return true?
Or do we need to give it a default value(verify's offset = 0x00) when we create the contract?

Reply:
`ContractConstants` ?

Reply:
> Or always return true?

I think that it should be false

Reply:
If it returns false, when some users send NEO or GAS to the contract by mistake, then they can't give it back.

If it has no bad influence when it returns true, we may set it true. 
If it has, we'd better to set it false.

Reply:
NEP-5 contracts should check whether the receiver is `payable`.

Reply:
> ContractConstants?

Creating a new file for only one constant?